core-web/libs/data-access/src/lib/dot-style-classes/dot-style-classes.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Observable } from 'rxjs';
```

Comment:
This is a very specific service for template so for now I think we need to keep it in that lib.